### PR TITLE
update `conda` in Jenkins environment

### PR DESCRIPTION
**Pull request by zacharyburnett**
This PR attempts to fix the issue in Jenkins jobs 
```
ModuleNotFoundError: No module named 'tqdm'
```

**Checklist**
- [ ] added entry in `CHANGES.rst` under the corresponding subsection
- [ ] updated relevant tests
- [ ] updated relevant documentation
- [ ] updated relevant milestone(s)
- [ ] added relevant label(s)

### CODE REVIEW

**Comment by zacharyburnett** [comment]
https://plwishmaster.stsci.edu:8081/job/RT/job/Roman-Developers-Pull-Requests/88/console
```
    Traceback (most recent call last):
      File "/users/iraf/build/boyle/workspace/RT/Roman-Developers-Pull-Requests/miniconda/lib/python3.7/site-packages/conda/exceptions.py", line 1079, in __call__
        return func(*args, **kwargs)
      File "/users/iraf/build/boyle/workspace/RT/Roman-Developers-Pull-Requests/miniconda/lib/python3.7/site-packages/conda/cli/main.py", line 84, in _main
        exit_code = do_call(args, p)
      File "/users/iraf/build/boyle/workspace/RT/Roman-Developers-Pull-Requests/miniconda/lib/python3.7/site-packages/conda/cli/conda_argparse.py", line 81, in do_call
        module = import_module(relative_mod, __name__.rsplit('.', 1)[0])
      File "/users/iraf/build/boyle/workspace/RT/Roman-Developers-Pull-Requests/miniconda/lib/python3.7/importlib/__init__.py", line 127, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
      File "<frozen importlib._bootstrap>", line 983, in _find_and_load
      File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
      File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
      File "<frozen importlib._bootstrap_external>", line 728, in exec_module
      File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
      File "/users/iraf/build/boyle/workspace/RT/Roman-Developers-Pull-Requests/miniconda/lib/python3.7/site-packages/conda/cli/main_create.py", line 10, in <module>
        from .install import install
      File "/users/iraf/build/boyle/workspace/RT/Roman-Developers-Pull-Requests/miniconda/lib/python3.7/site-packages/conda/cli/install.py", line 21, in <module>
        from ..core.solve import DepsModifier, Solver
      File "/users/iraf/build/boyle/workspace/RT/Roman-Developers-Pull-Requests/miniconda/lib/python3.7/site-packages/conda/core/solve.py", line 14, in <module>
        from .link import PrefixSetup, UnlinkLinkTransaction
      File "/users/iraf/build/boyle/workspace/RT/Roman-Developers-Pull-Requests/miniconda/lib/python3.7/site-packages/conda/core/link.py", line 43, in <module>
        from ..resolve import MatchSpec
      File "/users/iraf/build/boyle/workspace/RT/Roman-Developers-Pull-Requests/miniconda/lib/python3.7/site-packages/conda/resolve.py", line 9, in <module>
        from tqdm import tqdm
    ModuleNotFoundError: No module named 'tqdm'

`$ /users/iraf/build/boyle/workspace/RT/Roman-Developers-Pull-Requests/miniconda/bin/conda create -q -y -n tmp_env1 python=3.9 tqdm`
```

**Comment by zacharyburnett** [comment]
regtest running at https://plwishmaster.stsci.edu:8081/job/RT/job/Roman-Developers-Pull-Requests/91/console

File: JenkinsfileRT [hunk 1]
```diff
@@ -77,7 +77,7 @@ bc0 = new BuildConfig()
 bc0.nodetype = 'romancal'
 bc0.name = 'stable-deps'
 bc0.env_vars = env_vars
-bc0.conda_ver = '4.8.4'
```
Comment (zacharyburnett) [inline JenkinsfileRT:80]: updating `conda` appears to have resolved the issue

File: JenkinsfileRT [hunk 2]
```diff
@@ -77,7 +77,7 @@ bc0 = new BuildConfig()
...
+bc0.conda_ver = '22.11.1'
```
Comment (zacharyburnett) [inline JenkinsfileRT:80]: updating `conda` appears to have resolved the issue

**Review by ddavis-stsci — APPROVED** [review]
Looks good

**Comment by zacharyburnett** [comment]
https://plwishmaster.stsci.edu:8081/job/RT/job/Roman-Developers-Pull-Requests/91/testReport/
```
AttributeError: module 'crds.roman' has no attribute 'locate'
```

**Pull request merged**